requirements/requirements_doc.txt

```diff
@@ -12,4 +12,5 @@ sphinx-gallery==0.13.0
 sphinx-notfound-page==0.8.3
 sphinxcontrib-websupport==1.2.5
 sphinxemoji==0.2.0
+sphinx-toggleprompt
```

Comment:
We should specify the version, as in the other dependencies

The automated install used version 0.0.5 but that is a very old version from 3 years ago, maybe that is why it isn't working?

Reply:
Done. Thank you.